app/controllers/DataSetController.scala

```diff
@@ -57,7 +57,7 @@ class DataSetController @Inject()(userService: UserService,
       (__ \ 'sortingKey).readNullable[Long] and
       (__ \ 'isPublic).read[Boolean] and
       (__ \ 'tags).read[List[String]] and
-      (__ \ 'folderId).read[ObjectId]).tupled
+      (__ \ 'folderId).readNullable[ObjectId]).tupled
```

Comment:
PS: I just saw that the `folderId` is missing in the Swagger docs below (line 332)